addons/event/models/event_event.py

```diff
@@ -231,6 +232,7 @@ def _default_question_ids(self):
         compute_sudo=True)
     country_id = fields.Many2one(
         'res.country', 'Country', related='address_id.country_id', readonly=False, store=True)
+    event_url = fields.Char(string='Event URL', help="URL of the event if outside of the Odoo website")
```

Comment:
You can be more literal in helpers, maybe something along those lines:

'This URL can be set for online events, and will be used in links redirecting the attendees to the event. By default, the links redirect to the event page.'